components/postSightings/PostSighting.jsx

```diff
@@ -32,21 +32,23 @@ export default PostSighting = ({ navigation }) => {
   const windowHeight = useWindowDimensions().height;
 
   //Post sightings data to db
-  const Submit = () => {
-      console.log(globalUser)
+  const Submit = async () => {
+      // console.log(globalUser)
       let tempSightingData = {...sightingData}
       tempSightingData.created_at = new Date().toISOString()
       setSightingData({ bird: "", date_spotted: "", coordinates: "", user: globalUser.userId, sighting_img_url: ""})
-      console.log("submited", tempSightingData)
+      // console.log("submited", tempSightingData)
 
       // COOMENTED for dev purpososes
-      // addDoc(collection(db, "sightings"), tempSightingData)
-      // .then( docRef => {
-      //     // console.log(docRef)
-      // })
-      // .catch(err => {
-      //     console.log("Failed to submit a sighting")
-      // })
+      try {
+        const newSighting = await addDoc(collection(db, "sightings"), tempSightingData)
+        alert("Sighting successfully posted.")
+        navigation.navigate("SightingList")
```

Comment:
Maybe it could navigate to the new sighting post? Just a thought

Reply:
@Rim0Rad Can look into that on another ticket maybe? It would be a bit more work to get that sorted before tomorrow